server/config.py

```diff
@@ -50,6 +49,12 @@ def __init__(self):
         self.API_TOKEN_URI = "https://api.test.faforever.com/oauth/token"
         self.API_BASE_URL = "https://api.test.faforever.com/"
         self.USE_API = True
+        # Always set this key. It can be either the public key its self, or a
+        # path pointing to a pub key file.
+        self.API_JWT_PUBLIC_KEY = ""
```

Comment:
Might be worth adding it to `tests/data/test_conf.yaml` for documentation.
(That file is not used for regular tests, only those that check configuration refreshes work correctly, so it won't mess with the api_pub_key fixture.)

Reply:
Good idea